FirebaseRemoteConfig/Sources/RCNConfigFetch.m

```diff
@@ -218,6 +218,19 @@ - (void)refreshInstallationsTokenWithCompletionHandler:
                                                     NSLocalizedDescriptionKey : errorDescription
                                                   }]];
   }
+  // Check for projectID
+  if (!_options.projectID) {
```

Comment:
Why the conditions here and in FIRRemoteConfigComponent.m are different?

Reply:
removing the check here since we already check in `FIRRemoteConfigComponent`